libs/td/Makefile

```diff
@@ -0,0 +1,75 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=td
+PKG_VERSION:=1.8.0
+PKG_RELEASE:=$(AUTORELEASE)
+PKG_LICENSE:=Boost Software License - Version 1.0
```

Comment:
There are standardized license identifiers:
https://spdx.org/licenses/
The Boost library can be referenced with: `BSL-1.0`